tests/model_serving/model_server/raw_deployment/conftest.py

```diff
@@ -0,0 +1,45 @@
+import pytest
```

Comment:
please add proper label according to new raw kserve

Reply:
which ones? added if auth or route are needed